packages/genesys-spark-tokens/data/semantic/mode/dark.json

```diff
@@ -490,6 +490,20 @@
           "type": "color",
           "description": "The default background color for inputs."
         }
+      },
+      "search": {
+        "matching_text": {
+          "first_level": {
+            "value": "{theme.background.custom_accents.bold.120}",
```

Comment:
should these tokens have a `d` prefix at the end ?

Reply:
Not in this case, since the search match color is the same in light and dark